test/extended/builds/custom_build.go

```diff
@@ -0,0 +1,51 @@
+package builds
+
+import (
+	"fmt"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[Feature:Builds][Slow] custom build", func() {
```

Comment:
If we the goal of this e2e is to test that custom builds with buildah work, let's call it out here.
```suggestion
var _ = g.Describe("[Feature:Builds][Slow] custom build with buildah", func() {
```